src/site/stages/build/plugins/process-entry-names.js

```diff
@@ -97,8 +103,8 @@ function processEntryNames(buildOptions) {
           throw new Error(`Entry Name "${entryName}" was not found.`);
         }
 
-        // Link the element to the hashed entry name.
-        $el.attr(attribute, hashedEntryName);
+        // Link the element to the hashed entry name w/o the S3 bucket
+        $el.attr(attribute, `/${fileSearch}`);
```

Comment:
This is set to produce a /generated URL path. We have to do this because with webpack's publichPath set the file manifest generates full s3 paths with file hashes that will 404 when we try and run e2e/integration tests against them. The prearchive job in Jenkins still parses all the templates and updates the URLs to the correct bucket

Reply:
That makes sense 👍